app/push-sw.js

```diff
@@ -72,8 +72,18 @@ function notifyClient(obj) {
  * @return {Promise} Promise resolved once the notification is showed.
  */
 function showNotification(obj) {
-  const title = `Reminder for ${obj.recipient}`;
-  const body = obj.message;
+  let title;
+  if (!obj.recipients.length) {
+    title = 'Reminder';
+  } else if (obj.recipients.length === 1) {
+    title = `Reminder for ${obj.recipients[0]}`;
+  } else {
+    const firstRecipients = obj.recipients.slice(0, -1).join(', ');
+    const lastRecipient = obj.recipients[obj.recipients.length - 1];
+    title = `Reminder for ${firstRecipients} and ${lastRecipient}`;
+  }
```

Comment:
Super nit picking: Unlike French, English sentences have a comma before the "and" in a list of 3 or more elements. For instance:
> I have apples, bananas, and oranges.

http://www.grammarbook.com/punctuation/commas.asp (Rule 1)